stories/About/Developer Guide.mdx

```diff
@@ -0,0 +1,236 @@
+import { Meta } from "@storybook/blocks";
+import Image from "next/image";
+
+import supabase from '../assets/supabase_gridIron.png';
+import server from '../assets/supabase_server.png';
+import client from '../assets/client_component.png';
+  
+<Meta title="About/Developer Guide" />
+
+<div className="sb-container">
+  <div className='sb-section-title'>
+    <h1>Developer Guide</h1>
+  </div>
+ <section>
+<h3>Table of Contents</h3>
+      <ol>
+        <li><a href = "#started">Getting Started</a></li>
+        <li><a href = "#push">Pushing local branch to remote branch</a></li>
+        <li><a href = "#connect">Connecting to Supabase</a></li>
+        <li><a href = "#query">Query Supabase data from Next.js</a></li>
+      </ol>
+
+<h3 id = "started">Getting Started</h3>
+<ol>
+  <li>Clone the develop branch to your local machine:</li>
+  <li><code>git clone --single-branch --branch develop https://github.com/LetsGetTechnical/gridiron-survivor.git</code></li>
+  <li>Install the required packages with pnpm (NOT npm or yarn!)</li>
+  <li><code>pnpm install</code></li>
+  <li>Create a new branch for whatever you’re working on in the format firstname/feature-description, then check out your new branch. For example:</li>
+  <li><code>git branch richard/user-picks-page && git checkout richard/user-picks-page</code></li>
+  <li>Code your feature! 🤓</li>
+</ol>
+
+<h3 id = "push">Pushing local branch to remote branch</h3>
+<ol>
+<li>For the first push from local to remote, use --set-upstream origin along with the name of your local branch, like so:</li>
+<li><code>git push --set-upstream origin richard/user-picks-page</code></li>
+<li>Subsequent pushes will now go to your remote upstream branch.</li>
+</ol>
+
+<h3 id = "connect">Connecting to Supabase</h3>
+
+<ol>
+<li>In the project’s root folder, create an .env.local file:</li>
+<li><Image src = {supabase} alt = {""}/></li>
+<li>Place the following into your .env.local file:</li>
+<li><code>NEXT_PUBLIC_SUPABASE_URL=https://izumnfxsfrtbpyfptbeo.supabase.co</code></li>
+<li><pre><code>NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmF
+zZSIsInJlZiI6Iml6dW1uZnhzZnJ0YnB5ZnB0YmVvIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MDEzMTA0MjgsImV4cCI6MjAxNjg4N
+jQyOH0.hMNzXxIWrVMJjwwvC0tLuAwHGx8MzBXWIOTlkPTd-N8</code></pre></li>
+
+</ol>
+
+<h3 id = "query">Query Supabase data from Next.js</h3>
+<p>To create a Supabase client and query data from an Async Server Component, create a new page.tsx file like the example below.</p>
+<p>This example queries all entries stored in a hypothetical “notes” table from the database:</p>
+<Image src = {server} alt = {""}/>
+<p>Alternatively, using a Client Component:</p>
+<Image src = {client} alt = {""}/>
+</section>
+
+  <div class = "arw-container">
+    <a href="/?path=/docs/welcome--docs">Welcome</a>
+    <a href="/?path=/docs/about-sponsors--docs">Sponsors</a>
+  </div>
+
+</div>
+
+
+<style>
```

Comment:
@emestabillo These styles should live in its own SCSS file. Can you provide some direction or guidance on where this should live?